contracts/eosio.system/native.hpp

```diff
@@ -190,9 +190,9 @@ namespace eosiosystem {
          }
          
          ACTION( SystemAccount, canceldelay ) {
-            uint32_t   sender_id;
+            checksum256   trx_id;
```

Comment:
we should add a transaction_id_type typedef and use that for trx_id

Reply:
Missed it, will do.